README.md

```diff
@@ -1,148 +1,431 @@
-compatible like Upstart in Ubuntu 14.10):
+## Getting started guide
+So you successfully installed the script--great! 
 
-    sudo ln -s /usr/local/bin/mscs /etc/init.d/mscs
-    sudo update-rc.d mscs defaults
+At first, you probably want to [create a new world](#creating-a-new-world) or 
+[import an existing world](#importing-an-existing-world) into the script. 
 
-To manually add Bash Completion support:
+Then, you should adjust the [world properties](#adjusting-world-properties), 
+adjust the [global server settings](#adjusting-global-server-settings), and 
+enable any other 
+[server software](#enabling-forge-bungeecord-and-other-server-software) as needed.
 
-    sudo install -m 0644 mscs.completion /etc/bash_completion.d/mscs
+### Creating a new world
+The command to create a new world is:
 
-The Minecraft server software will be automatically downloaded to the
-following location on the first run:
+    mscs create [world] [port] <ip>
+  
+Where `world` is the name of the world you specify, 
+and `port` is the server port (by default, use 25565).
+`ip` is optional and will be used if you wish to create multiple worlds 
+across different servers. 
 
-    /opt/mscs/server/
+Afterwards, start the server via `mscs start [world]` where `world` 
+is the name of the world. The world will then shut down because you have to accept the EULA.
 
-### EULA
-As of Minecraft version 1.7.10, Mojang requires that users of their software read and agree to their [EULA](https://account.mojang.com/documents/minecraft_eula).  After you have read through the document, you need to modify the `eula.txt` file in your world's folder, changing the value of the `eula` variable from `false` to `true`.
+The EULA can be found in `/opt/mscs/worlds/myWorld` where `myWorld` 
+is the name given to the world you created. 
 
-    #By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula).
-    eula=true
+After accepting the EULA simply start the server using the same command above, 
+and you're all set!
 
-### Requirements
-We've made an attempt to utilize only features that are normally installed in
-most Linux and UNIX environments in this script, but there are a few
-requirements that this script has that may not already be in place:
-* Java JRE     - The Minecraft server software requires this.
-* Perl         - Most, if not all, Unix and Linux like systems have this
-                 preinstalled.
-* Python       - Required by the Minecraft Overviewer mapping software.
-* GNU Wget     - Allows the script to download software updates via the
-                 internet.
-* rdiff-backup - Allows the script to efficiently run backups.
-* Socat        - Allows the script to communicate with the Minecraft server.
-* Iptables     - Although not explicitly required, a good firewall should be
-                 installed.
+### Importing an existing world
+### Import Existing Worlds
```

Comment:
You have duplicate titles here.  The Index above links to `importing-an-existing-world`, but either wording is fine with me.